src/GitIgnoreChecker.php

```diff
@@ -56,7 +56,11 @@ public function getRepository() : Repository
      */
     public function isPathIgnored($path) : bool
     {
-        $relativePathToCheck = new RelativePath($this->getRepository(), $path);
+        try {
+            $relativePathToCheck = new RelativePath($this->getRepository(), $path);
+        } catch (InvalidArgumentException $e) {
+            return true;
```

Comment:
this should fail instead, because `new RelativePath` can be used only with really existing files/folders